docs-mslearn/framework/understand/ingestion.md

```diff
@@ -108,6 +108,21 @@ At this point, you have a data pipeline and are ingesting data into a central da
 
 <br>
 
+## Data Ingestion Metrics
+
```

Comment:
Add a simple paragraph to explain the table. Something simple like, "Consider the following key performance indicators (KPIs) as you mature data ingestion within your FinOps practice..."